packages/create-ripple-app/generators/updates/1.31.0/saofile.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
           data.dependencies['nuxt'] = `2.14.10`
           data.dependencies['@nuxtjs/gtm'] = '^2.4.0'
           data.dependencies['run-script-os'] = `^1.1.3`
-          data.devDependencies['babel-eslint'] = `^10.1.0`
+          data.devDependencies['@babel/eslint-parser'] = `^^7.12.1`
+          data.devDependencies['@babel/eslint-plugin'] = `^^7.12.1`
```

Comment:
I should revert this and create another file `1.32.0`

Reply:
yes, please

Reply:
I think we also need to remove `babel-eslint` from the project package.json